c++/src/kj/async-test.c++

```diff
@@ -733,6 +739,41 @@ TEST(Async, TaskSet) {
   EXPECT_EQ(1u, errorHandler.exceptionCount);
 }
 
+TEST(Async, LargeTaskSetDestruction) {
+  static constexpr size_t stackSize = 200 * 1024;
+
+  static auto testBody = [] {
+    EventLoop loop;
+    WaitScope waitScope(loop);
+    ErrorHandlerImpl errorHandler;
+    TaskSet tasks(errorHandler);
+
+    for (int i = 0; i < stackSize / sizeof(void*); i++) {
+      tasks.add(kj::NEVER_DONE);
+    }
+  };
+
+#if _WIN32
+  HANDLE thread = CreateThread(nullptr, stackSize, [](void*) -> DWORD {
```

Comment:
This is fine (assuming you can get CI to pass), but FWIW, another option might have been to use fibers. The fibers API lets you specify the stack size. (However you'd have to skip the test on platforms that don't support fibers.)

Reply:
Is there a macro for that? Skimming the code I don't see which platforms fibers are unavailable on.

Reply:
At present they are only missing on `__BIONIC__`, though #1208 suggests OpenBSD needs to be added, and I proposed there that we create a `KJ_USE_FIBERS` macro to encapsulate this and let people override their platform defaults... but I wouldn't do that in this PR obviously.

Reply:
I added `KJ_USE_FIBERS` to the top of the file so that it's there whenever `KJ_USE_FIBERS` is implemented in the codebase properly. It was a little unclear if fibers required exceptions but I assumed that that protection is in place in the test for a different reason.